src/main/scala/com/amazon/deequ/profiles/ColumnProfiler.scala

```diff
@@ -379,26 +402,81 @@ object ColumnProfiler {
         analyzer.column -> metric.value.get
       }
 
-    val knownTypes = schema.fields
-      .filter { column => columns.contains(column.name) }
-      .filter { _.dataType != StringType }
-      .map { field =>
-        val knownType = field.dataType match {
-          case ShortType | LongType | IntegerType => Integral
-          case DecimalType() | FloatType | DoubleType => Fractional
-          case BooleanType => Boolean
-          case TimestampType => String  // TODO We should have support for dates in deequ...
-          case _ =>
-            println(s"Unable to map type ${field.dataType}")
-            Unknown
-        }
+    val knownTypes =
```

Comment:
I think this can also be simplified, as it duplicates a lot of code

Reply:
Sure, will modify to avoid duplicating code.